components/net/filemanager_thread.rs

```diff
@@ -157,8 +159,6 @@ impl<UI: 'static + UIProvider> FileManager<UI> {
             FileManagerThreadMsg::DecRef(id, origin, sender) => {
                 if let Ok(id) = Uuid::parse_str(&id.0) {
                     spawn_named("dec ref".to_owned(), move || {
-                        // Since it is simple DecRef (possibly caused by close/drop),
```

Comment:
Because `DecRef` no has nothing to do with validity, the functionality is moved to `set_blob_url_validity`